lnwallet/fee_estimator.go

```diff
@@ -146,9 +157,20 @@ func (b *BtcdFeeEstimator) Start() error {
 		return err
 	}
 
-	// The fee rate is expressed in sat/KB, so we'll manually convert it to
-	// our desired sat/vbyte rate.
-	b.minFeeRate = SatPerVByte(relayFee / 1000)
+	// The fee rate is expressed in sat/kb, so we'll manually convert it to
+	// our desired sat/kw rate.
+	minRelayFeePerKw := SatPerVByte(relayFee / 1000).FeePerKWeight()
```

Comment:
This will lose some precision.

Reply:
Why can't we just convert directly to sat/kw and skip the vbyte hop?

Reply:
Fixed.